packages/shorebird_code_push_protocol/lib/src/models/app_capability.dart

```diff
@@ -0,0 +1,7 @@
+/// Operations that a user may or may not be able to perform to or in relation
+/// to an app.
+enum AppCapability {
+  /// Allows the user to create patches for an app at a non-100 rollout
+  /// percentage.
+  phasedPatchRollout,
```

Comment:
This will have an unstable serialization, since it will serialize as the enum name 🤷

Reply:
That's probably fine for now.